src/index.test.ts

```diff
@@ -63,6 +64,77 @@ describe("voices", () => {
 });
 
 describe("tts.websocket", () => {
+  let phonicWebSocket: PhonicWebSocket;
+
+  /**
+   * Promise that resolves with all messages received from the websocket after a period of no messages. Can be awaited
+   * multiple times.
+   */
+  let allMessagesReceived: Promise<PhonicWebSocketResponseMessage[]> | null =
+    null;
+  const maxIdleTime = 5000; // If we don't receive messages for 5 seconds, we consider that we received all of them.
+
+  const safeGet = (
+    messages: PhonicWebSocketResponseMessage[] | null,
+    idx: number,
+  ) => {
+    if (messages === null) {
+      throw new Error("messages is null");
+    }
+    if (messages[idx] === undefined) {
+      throw new Error(`Expected message at index ${idx} but received less`);
+    }
+    return messages[idx];
+  };
```

Comment:
@moroshko This `safeGet` function was the best way I could think of to avoid this boilerplate in each test function:
```typescript
if (message1 === undefined || message2 === undefined || message3 === undefined) {
  throw new Error(`Expected to get 3 messages, but got only ${messages.length}`);
}
```